Subaharish/HTML BASICS/assignment3.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Registration</title>
+    </head>
+    <body>
+        <h1>Register</h1>
+        <form>
+            First name:
+            <input type="text" name="Firstname"  placeholder="Firstname" required>
+            Last name:
+            <input type="text" name="Lastname" placeholder="Lastname" required>
+            <br>
+            <br>
+            Gender:
+
+            <label for="Male">Male:</label>
+            <input id="Male" type="radio" name="gender" value="male" checked>
+            <label for="Female">Female:</label>
+            <input id="Female" type="radio" name="gender" value="female">
+            <label for="Other">Other:</label>
+            <input id="Other" type="radio" name="gender" value="other" >
```

Comment:
`for` corresponds to `id`. Missing `id` attribute. Refer this: [HTML \<label> for Attribute - w3schools](https://www.w3schools.com/tags/att_label_for.asp)

Reply:
its there.. 
for both "other" and "female"

Reply:
??

Reply:
I don't know what is the mistake in this one

Reply:
??